### PR TITLE
Suffix prefix

### PR DESCRIPTION
**Pull request by titor-oopart**
Suffix and prefix method implemented

### CODE REVIEW

File: src/main/java/com/foundation/salesforce/core/utils/ValueAppender.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * with Jala Foundation.
+ */
+
+package com.foundation.salesforce.core.utils;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * ValueAppender class.
```
Comment (TheYisusGithub) [inline src/main/java/com/foundation/salesforce/core/utils/ValueAppender.java]: Please be as explicit when describing your class.

File: src/main/java/com/foundation/salesforce/core/utils/ValueAppender.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ *
+ * @author John Salazar Pinto.
+ * @version 1.0
```
Comment (TheYisusGithub) [inline src/main/java/com/foundation/salesforce/core/utils/ValueAppender.java]: Please put point at end to version.

**Review by TheYisusGithub — APPROVED** [review]

File: src/main/java/com/foundation/salesforce/core/utils/ValueAppender.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+    /**
+     * This method returns the the person that runs the scenaries.
+     *
+     * @return prefix.
+     */
+    public static String prefix() {
+        return prefix;
+    }
+}
```
Comment (Cxristian) [inline src/main/java/com/foundation/salesforce/core/utils/ValueAppender.java:49]: add one empty line

File: src/main/java/com/foundation/salesforce/core/utils/ValueAppender.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        Date actual = new Date();
+        SimpleDateFormat formatter = new SimpleDateFormat("dd-MM-yyyy HH:mm:ss");
+        suffix = " at " + formatter.format(actual);
+        return suffix;
+    }
+
+    /**
+     * This method returns the the person that runs the scenaries.
+     *
+     * @return prefix.
```
Comment (Cxristian) [inline src/main/java/com/foundation/salesforce/core/utils/ValueAppender.java]: add more documentation please

**Review by alszla — APPROVED** [review]

**Pull request merged**